canvas_modules/common-canvas/src/common-properties/controls/list/list.jsx

```diff
@@ -37,6 +37,7 @@ class ListControl extends AbstractTable {
 		super(props);
 		this.addRow = this.addRow.bind(this);
 		this.reactIntl = props.controller.getReactIntl();
+		this.readOnly = this.props.control.readOnly;
```

Comment:
Why have this?

Reply:
This was used to disable to `Add Value` button in list. I have removed this and have used `this.props.readOnly` in child component.

<img width="301" alt="Screenshot 2024-07-16 at 4 30 03 PM" src="https://github.com/user-attachments/assets/62c0895b-5ebf-447d-8791-7b24b67dfe2b">